common/util.c

```diff
@@ -13,6 +13,10 @@ int wrap(int i, int max) {
 	return ((i % max) + max) % max;
 }
 
+int max(int a, int b){
+	return a > b ? a : b;
+}
+
```

Comment:
Hmm, glibc already gives us a MAX macro. I think it comes from `#include <stdint.h>`. `tree/container.c` uses it.